### DISCUSSION TITLE
Add connectors to group

### DISCUSSION THREAD

**Discussion by skalten** [jgraph/drawio#3221] [Q&A]
Hi,
is there a way to add connectors to a group? I created a symbol for an amplifier as can be seen below.
![grafik](https://user-images.githubusercontent.com/93585789/204996399-dbf4e625-7b59-4bc2-8c0a-b3cbc9ec7112.png)

Now I want to connect a line to the ports on the left side which are themselves only lines. Since I grouped the whole thing I lost the connectors of the lines and can only use the ones of the triangle which are at the wrong positions. Is there a way to activate the connectors of the lines again or add them by hand to the group?

Thanks!

**Comment by skalten** [comment]
I found a workaround, but it is not very elegant.
I added an invisible rectangle in the background that aligns with the edges of the ports and included it into the group. Now I can add connectors that get anchored by holding down shift and placing them manually. The results are not very pretty since the connection is always slightly off and placing by hand is very cumbersome, but it works.
If anyone has a better solution please tell me.

**Comment by likwid101** [comment]
Can't you just right-click on the original image and edit connection points. The connection points are still useable when grouped. At least for me it is:


![image](https://user-images.githubusercontent.com/25729062/209905091-d3cc13b6-e070-4f80-9445-0d73aa2e19b7.png)
